docs/securing_client_connection.rst

```diff
@@ -258,3 +259,72 @@ On the client side, you have to provide ``ssl_cafile``, ``ssl_certfile``
 and ``ssl_keyfile`` on top of the other TLS/SSL configurations. See the
 :ref:`securing_client_connection:tls/ssl for hazelcast python clients`
 section for the details of these options.
+
+Username/Password Authentication
+================================
+
+You can protect your cluster using a username and password pair.
+In order to use it, enable it in member configuration:
+
+.. code:: xml
+
+    <security enabled="true">
+        <client-permissions>
+            ...
+        </client-permissions>
```

Comment:
We can remove these unnecessary parts

Reply:
OK: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/1b5f023e268221a1f09d54c8b47e1adbde7e57d1